src/main/scala/com/codacy/clangtidy/Main.scala

```diff
@@ -0,0 +1,15 @@
+package com.codacy.clangtidy
+
+object Main {
+
+  def main(args: Array[String]): Unit = {
+    // TODO make this dynamic
+    val toolName = "clang-tidy"
```

Comment:
can't we have this just in the docs? pattern.json?

Reply:
We can, but we're still not sure how this will be done.
Since this might be implemented as a "custom-tool" the name of the tool might change

Reply:
hmmm, this is weird